laravel/resources/views/pages/admin/report-departments-printable.blade.php

```diff
@@ -86,7 +86,7 @@
                         <td>{{ $slot->start_time }} ({{ date("h:i a", $start) }})</td>
                         <td>{{ $slot->end_time }} ({{ date("h:i a", $end) }})</td>
                         <td>
-                            @if(count($slot->user))
+                            @if($slot->user()->exists()))
```

Comment:
There's an extra parentheses in the template here. It causes the report to look like this:

![extra-parentheses](https://user-images.githubusercontent.com/1670549/47199842-c9a93280-d330-11e8-9fff-de3492ef6ebe.png)